roles/servicetelemetry/tasks/component_grafana.yml

```diff
@@ -115,3 +115,74 @@
         state: '{{ "present" if servicetelemetry_vars.graphing.enabled else "absent" }}'
         definition:
           '{{ ds_manifest }}'
+
+    - name: Load Cloud Overview Dashboard
+      k8s:
+        state: '{{ "present" if servicetelemetry_vars.graphing.grafana.dashboards.enabled else "absent" }}'
+        definition:
+          apiVersion: integreatly.org/v1alpha1
+          kind: GrafanaDashboard
+          metadata:
+            labels:
+              app: grafana
+              stf_owner: "{{ ansible_operator_meta.name }}"
+            name: rhos-cloud-dashboard-1
+            namespace: "{{ ansible_operator_meta.namespace }}"
+          spec:
+            name: rhos-cloud-dashboard.json
+            plugins:
+              - name: grafana-polystat-panel
+                version: "1.2.11"
```

Comment:
I thought about being fancy with a loop or whatever here and making this somewhat dynamic, but then I re-read the initial comment in https://issues.redhat.com/browse/STF-1623 which says, "Make sure not to overly complicate the management of the dashboards if at all possible."

None of the other dashboards need this plugin so I _could_ probably simplify this code path a bit with a loop, and maybe even some extra args or whatever with a loop over a hash, but I figured for the 4 dashboards here, it's just as easy to break them out and show them all.